common/app/common/configuration.scala

```diff
@@ -67,7 +67,7 @@ object GuardianConfiguration extends Logging {
   lazy val configuration = {
     // This is version number of the config file we read from s3,
     // increment this if you publish a new version of config
-    val s3ConfigVersion = 42
+    val s3ConfigVersion = 43
```

Comment:
I guess you changed the token in this v43

Reply:
yes, removed it all together